src/fr/uca/jgit/model/Commit.java

```diff
@@ -73,7 +73,7 @@ public static Commit loadCommit(String hash) {
 
         Commit newCommit = new Commit();
         try {
-            File myObj = new File(".git/logs/"+hash);
+            File myObj = new File(".jgit/logs/"+hash);
```

Comment:
Utilise ça pour écrire les paths
Path filePath = Paths.get(".jgit", "logs", this.hash());

Reply:
Je pars du principe que ce sera corrigé sur une autre branche car je m'occupe pas des commits donc je veux éviter un conflit inutile, mauvais call ?

Reply:
Alors tu enlèves la modifications d'après moi